pkg/fission-cli/flag/key/key.go

```diff
@@ -121,6 +121,7 @@ const (
 	EnvVersion         = "version"
 	EnvImagePullSecret = "imagepullsecret"
 	EnvExecutorType    = "executortype"
+	EnvIgnoreNotFound  = "ignoreNotFound"
```

Comment:
Lets define a single flag `IgnoreNotFound = "ignorenotfound"`
please note string should be "ignorenotfound". We won't be using camelcase in CLI options.